src/api/views.py

```diff
@@ -66,14 +67,11 @@ def check_domain_available(domain):
     a match.
     """
```

Comment:
**change before merging** Now that it throws an exception, the comment should indicate "throws RegistryError" to indicate that it has that potential